protocol/x/clob/keeper/process_operations.go

```diff
@@ -532,6 +532,22 @@ func (k Keeper) PersistMatchLiquidationToState(
 		return err
 	}
 
+	notionalQuoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
+		ctx,
+		matchLiquidation.PerpetualId,
+		new(big.Int).SetUint64(matchLiquidation.TotalSize),
+	)
+	if err != nil {
+		return err
+	}
+	notionalQuoteQuantums.Abs(notionalQuoteQuantums)
```

Comment:
Small suggestion, imo defining a new variable is more clear:

```suggestion
	absNotionalQuoteQuantums := new(big.Int).Abs(notionalQuoteQuantums)
```